core/templates/domain/topic/topic-object.model.ts

```diff
@@ -581,43 +581,51 @@ export class Topic {
     });
   }
 
-  // Reassigns all values within this topic to match the existing
+  // Reassigns all values within a copy topic variable to match this
```

Comment:
Creates a separate copy of this topic with the same values for the internal fields. This is performed ...

Reply:
Done